### PR TITLE
[Auction] Fade current refine settings.

### LINKED ISSUE #1367: [Auction] Artworks VC refinement status label should wrap before Refine button.
Given enough refinement options to generate a long status message, the following UI bug can be triggered:

img_7676

### PR DESCRIPTION
**Pull request by alloy**
Fixes #1367

Depends on https://github.com/artsy/eigen-artefacts/pull/3

- [x] Currently the alignment of the label to the button is still off.

### CODE REVIEW

File: Artsy/View_Controllers/Auction/AuctionViewController.swift [hunk 1]
```diff
@@ -232,7 +232,7 @@ extension AuctionViewController {
         activeModule = newModule as? ARSaleArtworkItemWidthDependentModule // Conditional cast always succeeds, but the compiler will complain otherwise.
 
         saleArtworksViewController.items = items
-        stickyHeader.subtitleLabel.text = saleViewModel.subtitleForRefineSettings(refineSettings, defaultRefineSettings: defaultRefineSettings())
+        stickyHeader.setSubtitleText(saleViewModel.subtitleForRefineSettings(refineSettings, defaultRefineSettings: defaultRefineSettings()))
```
Comment (alloy) [inline Artsy/View_Controllers/Auction/AuctionViewController.swift]: I gave up on UIScrollView + UILabel with dynamic width + Auto Layout.

File: Artsy/Views/Pages/ScrollingStickyHeaderView.swift [hunk 2]
```diff
@@ -56,10 +60,19 @@ class ScrollingStickyHeaderView: UIView {
             $0.constrainHeight("60")
         }
 
-        subtitleLabel.then {
+        subtitleScrollView.then {
+            $0.fadeAxis = .Horizontal
+            $0.showsHorizontalScrollIndicator = false
             self.addSubview($0)
-            self.leadingConstraints = $0.alignLeadingEdgeWithView(self, predicate: "20")
-            $0.alignBottomEdgeWithView(self, predicate: "-18")
```
Comment (alloy) [inline Artsy/Views/Pages/ScrollingStickyHeaderView.swift]: Ah… sigh, maybe the issues I was having with using AL had to do with me missing the negative sign.

Comment (orta) [inline Artsy/Views/Pages/ScrollingStickyHeaderView.swift]: aww

Comment (alloy) [inline Artsy/Views/Pages/ScrollingStickyHeaderView.swift]: Nope, doesn’t help one bit.

Comment (orta) [inline Artsy/Views/Pages/ScrollingStickyHeaderView.swift]: Oh well, if you make a branch with it, I can take a quick look next week?

**Comment by ArtsyMobile** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="true">There were <a href="https://eigen-ci.s3.amazonaws.com/3000/index.html">snapshot errors</a></td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="true">PR is classed as Work in Progress</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="true">Detected some Swift building time outliers</td>
    </tr>
  </tbody>
</table>

Time | Class | Function |
| --- | ----- | ----- |236.2ms | [AuctionViewController.swift](/artsy/eigen/tree/fade-filter-options/Artsy/View_Controllers/Auction/AuctionViewController.swift#L183) | @objc dynamic func showRefineTappedAnimated(animated: Bool)
187.3ms | [SwiftyJSON.swift](/artsy/eigen/tree/fade-filter-options/Pods/SwiftyJSON/Source/SwiftyJSON.swift#L445) | get {}
161.2ms | [WebSocket.swift](/artsy/eigen/tree/fade-filter-options/Pods/Starscream/Source/WebSocket.swift#L528) | private func processRawMessage(buffer: UnsafePointer<UInt8>, bufferLen: Int)
71.1ms | [Observable.swift](/artsy/eigen/tree/fade-filter-options/Pods/Interstellar/Sources/Observable.swift#L71) | private final func nextTokenHash() -> Int
67.9ms | [AuctionLotMetadataStackScrollView.swift](/artsy/eigen/tree/fade-filter-options/Artsy/View_Controllers/Live_Auctions/Views/AuctionLotMetadataStackScrollView.swift#L18) | required init(viewModel: LiveAuctionLotViewModelType, salesPerson: LiveAuctionsSalesPersonType, sideMargin: String)
66.8ms | [UIViewController+BlurredStatusView.swift](/artsy/eigen/tree/fade-filter-options/Artsy/View_Controllers/Core/UIViewController+BlurredStatusView.swift#L25) | func ar_presentBlurredOverlayWithTitle(title: String, subtitle: String, buttonState: BlurredStatusOverlayViewCloseButtonState = default)
66.5ms | [LiveAuctionsAdminViewController.swift](/artsy/eigen/tree/fade-filter-options/Artsy/View_Controllers/Live_Auctions/LiveAuctionsAdminViewController.swift#L22) | @objc override func viewDidLoad()
64.9ms | [UIViewController+BlurredStatusView.swift](/artsy/eigen/tree/fade-filter-options/Artsy/View_Controllers/Core/UIViewController+BlurredStatusView.swift#L31) | (closure)
59.3ms | [LiveAuctionLotViewController.swift](/artsy/eigen/tree/fade-filter-options/Artsy/View_Controllers/Live_Auctions/LiveAuctionLotViewController.swift#L49) | @objc override func viewDidLoad()
55.6ms | [RefinementOptionsViewController+Private.swift](/artsy/eigen/tree/fade-filter-options/Artsy/View_Controllers/Auction/RefinementOptionsViewController+Private.swift#L63) | func stackView() -> ORStackView


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://github.com/danger/danger/">danger</a>
</p>

File: Artsy/Views/Pages/ScrollingStickyHeaderView.swift (hunk 2 above)
Comment (alloy) [inline Artsy/Views/Pages/ScrollingStickyHeaderView.swift]: Never mind, I figured out a solution, I hate giving up.

**Comment by alloy** [comment]
fade refine settings label

**Comment by alloy** [comment]
Decided to do baseline alignement with the subtitle and button, which meant that a lot of snapshots were like a pixel off, but I think it’s better to understand than a magic number.

File: CHANGELOG.yml [hunk 3]
```diff
@@ -8,6 +8,8 @@ upcoming:
       - Parnter name shows on iPhone. - ash
       - New loading indicator. - ash
   user_facing:
+    live_actions:
```
Comment (ashfurrow) [inline CHANGELOG.yml]: I think `user_facing` is just a list and not a list of dictionaries, so `live_actions` can be taken out. If you're interested, feel free to correct that typo too :wink:

Comment (alloy) [inline CHANGELOG.yml]: Eh, I’ll be gone in a short bit, so I don’t want to be the one to break the build for such a pedantic issue ;)

**Comment by ashfurrow** [comment]
Looks good to merge!

**Comment by alloy** [comment]
Rebuilding without cache.

**Comment by alloy** [comment]
I don’t understand what it failed on. I can guarantee you that it’s green locally, so maybe just go ahead?

**Pull request merged**